finality-aleph/src/testing/mocks/validator_network.rs

```diff
@@ -0,0 +1,96 @@
+#![allow(dead_code)]
+use std::sync::Arc;
+
+use aleph_primitives::{AuthorityId, KEY_TYPE};
+use sp_keystore::{testing::KeyStore, CryptoStore};
+
+use crate::{
+    network::{mock::Channel, Data, Multiaddress, NetworkIdentity, PeerId},
+    validator_network::Network,
+};
+
+type MockPeerId = AuthorityId;
```

Comment:
if possible use some primitive type like u32,string or smth like this

Reply:
not blocking for this pr, but wouldn't we like this new network part to be not dependent on any extern types (Especially ones from substrate)? @timorl

Reply:
That might be nice, but it would require lots of traits for keys which we don't yet have, and writing them would take a lot of time. We probably should have a parkinglot task for improving that.

Reply:
Added as 1454. @lesniak43 since you were also a proponent of that iirc and probably want to know.